chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -947,3 +947,90 @@ Example
 - Check WMS with Swiss ech-0056 profile (html): `https://api3.geo.admin.ch/owschecker/form?base_url=http%3A%2F%2Fwms.zh.ch%2Fupwms&service=WMS <../../../owschecker/form?base_url=http%3A%2F%2Fwms.zh.ch%2Fupwms&service=WMS>`_
 - Check WMTS with Swiss ech-0056 profile (xml): `https://api3.geo.admin.ch/owschecker/bykvp?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS <../../../owschecker/bykvp?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS>`_
 - Check WMTS with Swiss ech-0056 profile (html): `https://api3.geo.admin.ch/owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS <../../../owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS>`_
+
+
+
+
+.. _owschecker_description:
+
+----------
+
+Stationboard
+------------
+
+This service provide the next connections from a location to a specific destination
+
+URL
+***
+
+::
+
+  https://api3.geo.admin.ch/stationboard/stops/{id}
+
+Input Parameters
+****************
+
++-----------------------------------+-------------------------------------------------------------------------------------------+
+| Parameters                        | Description                                                                               |
++===================================+===========================================================================================+
+| **destination (required)**        | Specifies the arrival location of the connection                                          |
++-----------------------------------+-------------------------------------------------------------------------------------------+
+
+Examples
+********
+
+- Get the fives next connection from a location to all destinations: `https://api3.geo.admin.ch/stationboard/stops/8501120?destination=all <../../../stationboard/stops/8501120?destination=all>`_
+- Get the fives next connection from a location to one destination: `https://api3.geo.admin.ch/stationboard/stops/8501120?destination=Genève <../../../stationboard/stops/8501120?destination=Gen%C3%A8ve>`_
+
+Response syntax
+***************
+
+Here is an example of response.
+
+.. code-block:: javascript
+
+    {
+        "via": "Lausanne; Renens VD; Morges; Allaman; Rolle; Gland; Nyon; Coppet",
+        "d": "20/11/2015 11:21",
```

Comment:
Hard to understand what `d` means here. Can't we use a better name for this property?